<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IBM681-R9WCGFW" name="AutomationTest03" tests="1" failures="1" timestamp="22 Aug 2017 09:47:58 GMT" time="3.279" errors="0">
  <testcase name="@AfterMethod afterTestMethod" time="0.0" classname="com.ibm.test.TestCase02">
    <failure type="org.openqa.selenium.NoSuchWindowException" message="Unable to get browser (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 16 milliseconds
Build info: version: &amp;apos;unknown&amp;apos;, revision: &amp;apos;5234b32&amp;apos;, time: &amp;apos;2017-03-10 09:00:17 -0800&amp;apos;
System info: host: &amp;apos;IBM681-R9WCGFW&amp;apos;, ip: &amp;apos;9.195.65.247&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.8.0_144&amp;apos;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{se:ieOptions={browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, ie.useLegacyFileUploadDialogHandling=false, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, initialBrowserUrl=http://localhost:40886/, ignoreProtectedModeSettings=true, enableElementCacheCleanup=true}, browserName=internet explorer, pageLoadStrategy=normal, javascriptEnabled=true, version=11, platform=WINDOWS, unexpectedAlertBehaviour=dismiss}]
Session ID: 0f88b846-ffea-43e2-8078-349ac556135e">
      <![CDATA[org.openqa.selenium.NoSuchWindowException: Unable to get browser (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 16 milliseconds
Build info: version: 'unknown', revision: '5234b32', time: '2017-03-10 09:00:17 -0800'
System info: host: 'IBM681-R9WCGFW', ip: '9.195.65.247', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_144'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{se:ieOptions={browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, ie.useLegacyFileUploadDialogHandling=false, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, initialBrowserUrl=http://localhost:40886/, ignoreProtectedModeSettings=true, enableElementCacheCleanup=true}, browserName=internet explorer, pageLoadStrategy=normal, javascriptEnabled=true, version=11, platform=WINDOWS, unexpectedAlertBehaviour=dismiss}]
Session ID: 0f88b846-ffea-43e2-8078-349ac556135e
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:638)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:661)
	at org.openqa.selenium.ie.InternetExplorerDriver.getScreenshotAs(InternetExplorerDriver.java:192)
	at com.ibm.utils.TestBase.getScreenShotOnFailure(TestBase.java:459)
	at com.ibm.test.TestCase02.afterTestMethod(TestCase02.java:58)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterTestMethod -->
  <testcase name="@AfterClass rollOff" time="0.0" classname="com.ibm.test.TestCase02">
    <skipped/>
  </testcase> <!-- @AfterClass rollOff -->
  <testcase name="testCase02_VerifyUserIsAbleToSearchStore" time="0.312" classname="com.ibm.test.TestCase02">
    <failure type="java.lang.AssertionError" message="Test Case Marked Failed Due to Exception expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test Case Marked Failed Due to Exception expected [true] but found [false]
	at com.ibm.test.TestCase02.testCase02_VerifyUserIsAbleToSearchStore(TestCase02.java:101)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testCase02_VerifyUserIsAbleToSearchStore -->
</testsuite> <!-- AutomationTest03 -->
